homework8_max_3.py

```diff
@@ -0,0 +1,29 @@
+"""
+Homework8.
+
+Description:
+    Given the list of integers ( positive , negative, zeros)
+    Find max multiplication of three values in list
+    l1 = [-7, 1, 2, 0, 3, 12, -24, 5, -1, 6]
+
+    Input: [-1, 1, 2, 0, 3, 12, 4, 5, -1, 6]
+    Output: Max value = "360". Nums are: (12, 5, 6)
+
+    Input: [-7, 1, 2, 0, 3, 12, -24, 5, -1, 6]
+    Output: Max value = "2016". Nums are: (-7, 12, -24)
+
+"""
+
+l1 = [-7, 1, 2, 0, 3, 12, -24, 5, -1, 6]
+
+# create a dict where key = multiplication and value = (a, b, c)
+d = {}
+for a in range(len(l1)):
```

Comment:
Correct solution. But complexity = O(n^3)
Such solution acceptable for junior level only.

Please pay attention to sorted list [-24, -7, -1, 0, 1, 2, 3, 5, 6, 12] here possible options to get max are 
el[0] * el[1]* el[-1]  or el[-1] * el[-2]* el[-3]   so you need just to compare which group gives max. And that's it )
Complexity = (O(1) + O(1) +O(1) )*2 = O(1)